packages/x-charts/src/RadarChart/RadarDataProvider/radar.types.ts

```diff
@@ -0,0 +1,37 @@
+export interface MetricConfig {
+  /**
+   * The name of the metric.
+   */
+  name: string;
+  /**
+   * The minimal value of the domain.
+   * @default 0
+   */
+  min?: number;
+  /**
+   * The maximal value of the domain.
+   * If not provided, it gets computed to display the entire chart data.
+   */
+  max?: number;
+}
+
+export interface RadarConfig {
+  /**
+   * The different metrics shown by radar.
```

Comment:
This is more of a nit, but I'm not sure "different" is adding much here. 
```suggestion
   * The metrics shown by radar.
```